ci/test-flow-collector.sh

```diff
@@ -0,0 +1,66 @@
+set -eu
+
```

Comment:
@zyiou Why is this added in CI? And not as an e2e test?

Reply:
That is suggested by Antonin that we want to create script like `test-conformance-gke.sh`-- can be run locally and also daily CI to validate the manifest.

Reply:
It makes sense under ci/ IMO, since this is not a Golang test.

Reply:
I was thinking of other option of turning it into golang test and run as part of e2e suite if possible. Running it only once daily makes sense as the main purpose is  to check elasticstack manifest.